src/it/junit/issue_19-multiple-execution/src/test/resources/features/feature2.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Feature1
```

Comment:
right, I am just saying its a little confusing since this is the "feature 2" file but we called it "feature 1" in the feature file itself. Just saying we should be consistent